pytorch_lightning/loops/base.py

```diff
@@ -99,6 +99,53 @@ def connect(self, **kwargs: "Loop") -> None:
         Linked loops should form a tree.
         """
 
+    def replace(self, **loops: Union["Loop", Type["Loop"]]) -> None:
+        """Optionally replace one or multiple of this loop's sub-loops.
+
+        This methods takes care of instantiating the class (if necessary) with all existing arguments, connecting all
+        sub-loops of the old loop to the new instance, setting the ``Trainer`` reference, and conneting the new loop to
+        the parent.
+
+        Args:
+            **loops: A ``Loop`` subclass or instance. The name used should match the loop attribute name you want to
```

Comment:
here it is meant the name of the argument

Reply:
yes, I mean to suggest making it clear the name mentioned here refers to the name of the argument. 

LGTM